src/GlobalAccount.ts

```diff
@@ -35,6 +37,149 @@ export function setGlobalNetworkAccount(account: AccountDB.AccountCopy): void {
   cachedGlobalNetworkAccountHash = account.hash
 }
 
+interface NetworkConfigChanges {
+  cycle: number
+  change: any
+  appData: any
+}
+
+export const updateGlobalNetworkAccount = async (cycleNumber: number): Promise<void> => {
+  if (!cachedGlobalNetworkAccountHash) return
+  const networkAccount = rfdc()(cachedGlobalNetworkAccount)
+  const changes = networkAccount.data.listOfChanges as NetworkConfigChanges[]
+  if (!changes || !Array.isArray(changes)) {
+    return
+  }
+  const activeConfigChanges = new Set<string>()
+  for (const change of changes) {
+    // skip future changes
+    if (change.cycle > cycleNumber) {
+      continue
+    }
+    const changeHash = Crypto.hashObj(change)
+    // skip handled changes
+    if (appliedConfigChanges.has(changeHash)) {
+      activeConfigChanges.add(changeHash)
```

Comment:
Mentioned in core PR as well.. Should we be adding activeConfigChanges here.  The comment implied to skip handled changes